<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string> 文件(&amp;F)</string>
    </property>
    <addaction name="newFileAct"/>
    <addaction name="openFileAct"/>
    <addaction name="savaFileAct"/>
    <addaction name="saveAsAct"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutAct"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openFileAct">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="savaFileAct">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="saveAsAct">
   <property name="text">
    <string>另存为(&amp;save as)</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="newFileAct">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
